client/app/components/ParameterValueInput.jsx

```diff
@@ -59,7 +59,7 @@ class ParameterValueInput extends React.Component {
   }
 
   onSelect = (value) => {
-    const isDirty = !isEqual(value, this.props.value);
+    const isDirty = !isEqual(trim(value), trim(this.props.value));
```

Comment:
Before:
<img width=200 src="https://user-images.githubusercontent.com/486954/68008020-16cf1c80-fc86-11e9-820d-20f89e30903c.gif" />

After:
<img width=200 src="https://user-images.githubusercontent.com/486954/68008023-1c2c6700-fc86-11e9-8cf3-efd12610a807.gif" />

Reply:
I wonder if there is a legitimate use case for multiple spaces in a parameter :grimacing: